Makefile

```diff
@@ -289,6 +295,7 @@ TM_LIBS				= third_party/proto/tendermint/libs/bits
 
 GOGO_PROTO_TYPES    = third_party/proto/gogoproto
 COSMOS_PROTO_TYPES  = third_party/proto/cosmos_proto
+GOOGLE_PROTO_TYPES  = third_party/proto/google/api
```

Comment:
Please rebase this PR to latest master as we no longer use third_party dir and HTTP to download proto files, instead use modvendor to sync them into vendor dir

Reply:
Proto files are not generating in vendor directory using modvendor. I tried by rebasing to master and proto files are not generating. Even tested in master branch directly and facing same issue there too.

Reply:
does #818 help?

Reply:
I didn't notice this commit. Just now tested in master and its working. Will try same by rebasing this PR to master. Thanks @boz

Reply:
@troian `github.com/googleapis/googleapis/google/api` is a non go repository. So we cannot use `go mod vendor` and sync proto files here. Is there any other way we can solve this?

Reply:
@akhilkumarpilli this way should be ok https://github.com/ovrclk/akash/pull/819